ztp/policygenerator/policyGen/policyHelper.go

```diff
@@ -30,19 +30,49 @@ func CreatePolicySubject(policyName string) utils.Subject {
 	return subject
 }
 
-func CreatePlacementRule(name string, namespace string, matchKeyValue map[string]string) utils.PlacementRule {
+func CheckBindingRules(pgtName string,
+	bindingRules map[string]string, bindingExcludedRules map[string]string) error {
+
+	for key := range bindingRules {
```

Comment:
this logic is strange , I can have binding rule like;
- key: "du-sno"
  operator: "Exists"

And exclude binding rule like;
- key: "du-sno"
  operator: NotIn
  value: [east] 

Just check the same set not both.

Reply:
Thanks for pointering this out. Right, this is the only valid case for same key in bindingRules & bindingExcludedRules. I have updated the PR to address this.